### PR TITLE
BETA: Register tawsh.is-a.dev

### PR DESCRIPTION
**Pull request by dev-tosh**
Added `tawsh.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Review by wdhdev — CHANGES_REQUESTED** [review]
I cannot reach your website.

File: domains/antonybahati.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "shellcodegroup",
+           "email": "shellcodegroup@gmail.com",
+           "discord": "1197076751390625825"
+        },
+    
+        "record": {
+            "CNAME": "shellcodegroup.github.io"
```
Comment (DEV-DIBSTER) [inline domains/antonybahati.json:9]: 404 GitHub. No content displayed.

File: domains/tawsh.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "shellcodegroup",
+        "email": "shellcodegroup@gmail.com"
+    },
+    "record": {
+        "CNAME": "shellcodegroup.github.io"
```
Comment (DEV-DIBSTER) [inline domains/tawsh.json:7]: 404 GitHub. No content displayed.

File: domains/toni.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "shellcodegroup",
+           "email": "shellcodegroup@gmail.com",
+           "discord": "1197076751390625825"
+        },
+    
+        "record": {
+            "CNAME": "shellcodegroup.github.io"
```
Comment (DEV-DIBSTER) [inline domains/toni.json:9]: 404 GitHub. No content displayed.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Pull request closed without merging**